internal/admin/templates/layout.html

```
@@ -156,7 +156,19 @@
         Rendered only when the operator has configured an upstream — the
         whole group is absent on a bridge with none. */ -}}
     {{if .UPnPSources}}
-    <p class="nav-group">UPnP</p>
+    {{- /* The group heading carries the way to MANAGE these servers.
+        Without it the sidebar names two different UPnP things — this
+        group, which browses an upstream's music, and Server > UPnP, which
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)